Core/ResponseMapper.php

```diff
@@ -1,4 +1,11 @@
 <?php
+/**
```

Comment:
Because the HTML returned by the SDK is not the same as the HTML we produce in the view. We would also like to keep all HTML in the views and not in the controllers.